### DISCUSSION TITLE
Adjust title of "mealie" to that of the group

### DISCUSSION THREAD

**Discussion by RealFoxie** [mealie-recipes/mealie#1978] [Feature Request]
Every instance of Mealie will have the word "Mealie" at the top (in the `v-toolbar title`).
To more easily differentiate what group you are currently viewing, it would make sense to change this to the name of the group. It also makes your instance feel more personal.

**Comment by imRadwolf** [comment]
I also tried searching for a way to simply just edit the text displayed to no avail
